ui/src/styles/typography.js

```diff
@@ -0,0 +1,25 @@
+import styled from "styled-components";
+import colours from "./colours";
+import { device } from "./device";
+
+const extraLargeText = "42px";
+const largeText = "36px";
+const mediumText = "16px";
+const smallText = "14px";
+const extraSmallText = "12px";
+const tinyText = "10px";
+
+const TitleText = styled.p`
+  font-size: ${largeText};
+  font-weight: 700;
+  text-align: center;
+  color: ${colours.accentColour};
+
+  @media ${device.mobileS} {
+    font-size: ${smallText};
+  }
+`;
+
+export const typography = {
+    TitleText: TitleText,
```

Comment:
Hmm why did we decide to export an object this way?

I'd rather just export each component separately and then import the component from `styles/typography`. This works better with tree shaking btw.

Reply:
Only reason why is we can easily change the typography of something within the file. What's currently implemented is a bad example but if we had: 
`export const typography = {
    TitleText: LargeBlackOpenSans,
}`
 and wanted to change the style of the title, we could just: 
`export const typography = {
    TitleText: LargeBlueInter,
}`
versus having to change all occurrences in the file. Having the exports this way enables us to componentize. 

Buttttt at the same time I do agree that it has its cons.

Reply:
Ah gotcha, I can see the reasoning then.

I'm of the mind that this project won't get big enough so we probably don't need to prematurely optimize this stuff, even if that means having to find-and-replace like 5-6 component names, but your call, feel free to keep as is.

Reply:
Hmm I agree, I'll change it!